telethon/telegram_client.py

```diff
@@ -2470,4 +2471,75 @@ def get_input_entity(self, peer):
             'Make sure you have encountered this peer before.'.format(peer)
         )
 
+    def edit_2FA(self, current_password=None, new_password=None, hint='', email=None):
```

Comment:
Python doesn't put methods on upper case at all. You don't do things like `SomeThing`, you do `some_thing`, so there's no need to do `2FA`, just `2fa`.